agent/Makefile

```diff
@@ -38,10 +38,22 @@ compile-crane: ##
 	GOOS=linux GOARCH=amd64 CGO_ENABLED=0 go build ${LDFLAGS} -o ../../build/out/crane; \
 	cd -
 
+.PHONY: compile-crane-windows
+compile-crane-windows: ##
+	cd cmd/crane && \
+	GOOS=windows GOARCH=amd64 CGO_ENABLED=0 go build ${LDFLAGS} -o ../../build/out/crane.exe; \
+	cd -
+
 .PHONY: compile-dagent
 compile-dagent: ##
 	cd cmd/dagent && \
-	GOOS=linux GOARCH=amd64 CGO_ENABLED=0 go build ${LDFLAGS} -o ../../build/out/dagent; \
+	GOOS=windows GOARCH=amd64 CGO_ENABLED=0 go build ${LDFLAGS} -o ../../build/out/dagent.exe; \
+	cd -
+
+.PHONY: compile-dagent-windows
+compile-dagent-windows: ##
```

Comment:
Instead of duplication manipulate variables instead, same could be done as with image_version, defaults to linux, but on machines you can do `goos=windows make compile-dagent`.